python_modules/libraries/dagster-snowflake/dagster_snowflake/snowflake_io_manager.py

```diff
@@ -128,9 +129,29 @@ def my_table_a(my_table: pd.DataFrame) -> pd.DataFrame:
                 ),
                 is_required=False,
             ),
+            "time_data_to_string": Field(
```

Comment:
`store_datetimes_without_timezones_as_strings` might be clearer? also, I think worth mentioning in the docs that the reason we offer this is to offer compatibility with earlier versions.

Reply:
`store_timestamps_as_strings` sounds good